app/src/googlevrFlat/java/org/mozilla/vrbrowser/BrowserActivity.java

```diff
@@ -173,7 +173,7 @@ private void loadFromIntent(final Intent intent) {
             final MyGeckoViewPermission permission = new MyGeckoViewPermission();
             permission.androidPermissionRequestCode = REQUEST_PERMISSIONS;
             mGeckoSession.setPermissionDelegate(permission);
-            uriValue = (uri != null ? uri.toString() : SessionStore.DEFAULT_URL);
+            uriValue = (uri != null ? uri.toString() : SessionStore.get().getHomeUri());
```

Comment:
Yep, we will never get a region upon first run after install unless we use something like postMessage. 

You will get updated region info after the geolocation request finishes in subsequest home reloads though (after adding the setRegion call to onGeolocationRequestSuccess)

Reply:
filed #409 - thank you!